src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -99,113 +82,80 @@ static void mpu6050FindRevision(gyroDev_t *gyro)
         }
     }
 }
+*/
 
 /*
  * Gyro interrupt service routine
  */
 #if defined(USE_MPU_DATA_READY_SIGNAL) && defined(USE_EXTI)
 static void mpuIntExtiHandler(extiCallbackRec_t *cb)
 {
-#ifdef DEBUG_MPU_DATA_READY_INTERRUPT
-    static uint32_t lastCalledAtUs = 0;
-    const uint32_t nowUs = micros();
-    debug[0] = (uint16_t)(nowUs - lastCalledAtUs);
-    lastCalledAtUs = nowUs;
-#endif
     gyroDev_t *gyro = container_of(cb, gyroDev_t, exti);
     gyro->dataReady = true;
     if (gyro->updateFn) {
         gyro->updateFn(gyro);
     }
-#ifdef DEBUG_MPU_DATA_READY_INTERRUPT
-    const uint32_t now2Us = micros();
-    debug[1] = (uint16_t)(now2Us - nowUs);
-#endif
 }
 #endif
 
-static void mpuIntExtiInit(gyroDev_t *gyro)
+void mpuIntExtiInit(gyroDev_t *gyro)
 {
-    if (!gyro->mpuIntExtiConfig) {
+    if (!gyro->dev->irqPin) {
```

Comment:
It's not clear to me that the `irqPin` should be part of the `busDevice_t` structure. I think it should probably be part of the `gyroDev_t` structure.